fullscreen.bs

```diff
@@ -505,9 +516,16 @@ properties apply to this pseudo-element either.
 <p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
 <a>element</a> that has its <a>fullscreen flag</a> set.
 
-<p class="note no-backref">This makes it different from the {{Document/fullscreenElement}} API,
+<p class="note no-backref">This makes it different from the
+{{DocumentOrShadowRoot/fullscreenElement}} API,
 which returns a <a>document</a>'s <a>fullscreen element</a>.
 
+For each <a>node tree</a>, if the result of <a>retargeting</a> its <a>shadow-including root</a>'s
```

Comment:
This is about making the shadow host also match the `:fullscreen` selector, right? The shadow host isn't being automatically added to the top layer, so it won't have a `::backdrop` pseudo-element and would never be visible unless the fullscreen element has a transparent background.

In this revision of this PR, shadow trees are handled quite unlike iframes, and I think that actually turned out better, so can we take it even further?

(Not sure how to implement this otherwise, actually doing this as part of selector matching would be too slow.)